### ISSUE TITLE
Add instructions for how to install Chocolatey offline from a local nupkg

### ISSUE THREAD

**Issue by TheCakeIsNaOH** [chocolatey/docs#395]
## Is Your Feature Request Related To A Problem? Please describe.

There have been users that want to install Chocolatey from a local, predownloaded `.nupkg`. There are not currently instructions on how to do that.

## Describe The Solution. Why is it needed?

Add install instructions for how to install Chocolatey from a local nupkg. It may be good to also create a guide for how to install both Chocolatey, and packages from predownloaded nupkgs, for users that want an offline setup without requiring a full on nexus or similar repository.

## Additional Context.

There is a PR that updates the `install.ps1` script to work with local `.nupkg` files. This documentaion would require that PR to be merged first, because the instructions would presumably use the updated `install.ps1` script.

## Related Issues

(From @corbob)
https://gitlab.com/chocolatey/community-infrastructure/community.chocolatey.org/-/merge_requests/1129

**Comment by JPRuskin** [comment]
These exist, but could/should be improved: https://docs.chocolatey.org/en-us/choco/setup#completely-offline-install

**Comment by TheCakeIsNaOH** [comment]
That set of instructions is for people that have a internal repository setup, not for installing from a `.nupkg` on the local filesystem. But if it is updated to work with both a local `.nupkg` and from an internal repository, that is fine with me.

**Comment by JPRuskin** [comment]
Yeah, I'd be happy to leave a set of "install from local repository" instructions but will admit that when first reading those I thought "Wait, completely offline and yet we can access an internal repository?" (i.e. misinterpreted it as not connected to a network rather than not connected to the internet)
